Source/Pads/Gamepad.as

```diff
@@ -108,38 +147,39 @@ class DashboardPadGamepad : IDashboardPad
 			nvg::Fill();
 			nvg::ResetScissor();
 		}
+		float fillAlphaDown = pedalBrake > 0.1f ? 1.0f : Setting_Gamepad_OffAlpha;
+		if (Setting_Gamepad_UseBorderGradient) {
+			nvg::StrokePaint(Setting_Gamepad_BorderGradient.GetPaint(vec2(), size, fillAlphaDown));
+		} else {
+			nvg::StrokeColor(Setting_Gamepad_BorderColor);
+		}
 		nvg::Stroke();
 
 		// Up & Down texts
 		if (Setting_Gamepad_UpDownSymbols) {
 			nvg::BeginPath();
-			nvg::FontFace(g_font);
-			nvg::FontSize(midSize / 2);
-			nvg::FillColor(Setting_Gamepad_TextColor);
+			nvg::FontFace(m_font);
+			nvg::FontSize(Setting_Gamepad_FontSize * 1.5f);
 			nvg::TextAlign(nvg::Align::Middle | nvg::Align::Center);
+			nvg::FillColor(WithAlpha(Setting_Gamepad_FontColor, fillAlphaUp));
 			nvg::TextBox(midX, topSize / 2, midSize, Icons::AngleUp);
+			nvg::FillColor(WithAlpha(Setting_Gamepad_FontColor, fillAlphaDown));
 			nvg::TextBox(midX, bottomY + bottomSize / 2, midSize, Icons::AngleDown);
 		}
 
 		// Steering percentage
 		if (Setting_Gamepad_SteerPercentage) {
-			nvg::FontFace(g_font);
-			nvg::FontSize(Setting_Gamepad_SteerPercentageSize);
-			nvg::FillColor(Setting_Gamepad_TextColor);
-
-			// Left
-			if (steerLeft > 0) {
-				nvg::BeginPath();
-				nvg::TextAlign(nvg::Align::Middle | nvg::Align::Right);
-				nvg::TextBox(0, size.y / 2, leftSize - Setting_Gamepad_Spacing, tostring(Math::Round(steerLeft * 100)) + "%");
-			}
-
-			// Right
-			if (steerRight > 0) {
-				nvg::BeginPath();
-				nvg::TextAlign(nvg::Align::Middle | nvg::Align::Left);
-				nvg::TextBox(rightX + Setting_Gamepad_Spacing, size.y / 2, rightSize, tostring(Math::Round(steerRight * 100)) + "%");
-			}
+			RenderSteerPercentage(
+				size.y / 2.0f,
+				Setting_Gamepad_Spacing,
+				steerLeft,
+				fillAlphaLeft,
+				leftSize,
+				steerRight,
+				fillAlphaRight,
+				rightX,
+				rightSize
+			);
```

Comment:
This function call (and every call to this function) is not very readable. Can we fix this?